conf/pulsar_tools_env.sh

```diff
@@ -42,7 +42,8 @@
 # PULSAR_GLOBAL_ZK_CONF=
 
 # Extra options to be passed to the jvm
-PULSAR_MEM="-Xmx128m -XX:MaxDirectMemorySize=128m"
+# PULSAR_MEM="-Xmx128m -XX:MaxDirectMemorySize=128m"
+PULSAR_MEM="-Xmx4g -XX:MaxDirectMemorySize=128m"
```

Comment:
is this change necessary?

Reply:
No, I had to add that in order to be able to submit the NAR I was testing with.